controllers/chat/sendSignedMessage.js

```diff
@@ -90,7 +90,7 @@ const sendSignedMesage = async (req, res) => {
     let channelMember = [];
     let senderInfo = [];
     // Prevent if user try to chat with blocked user
-    let channelType = createdChannel?.channel?.channel_type;
+    let channelType = createdChannel?.channel?.channel_type || req.body.channelType || 0;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here is a good approach to handle the case when `channelType` is not defined in the `createdChannel` object. However, it's important to validate the `req.body.channelType` before using it as a fallback. This is because if an attacker sends a non-integer value for `channelType`, it could potentially lead to unexpected behavior or errors in the subsequent code that uses `channelType`. I suggest adding a check to ensure `req.body.channelType` is an integer.

```diff
- let channelType = createdChannel?.channel?.channel_type || req.body.channelType || 0;
+ let channelType = createdChannel?.channel?.channel_type || (Number.isInteger(req.body.channelType) ? req.body.channelType : 0);
```
This way, we only use `req.body.channelType` if it's a valid integer, otherwise, we default to `0`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->